frontend/src/cards/MapCard.tsx

```diff
@@ -55,21 +53,33 @@ function MapCardWithKey(props: MapCardProps) {
     },
   };
 
-  // TODO - make sure the legends are all the same
-  const [breakdownFilter, setBreakdownFilter] = useState<string>("");
-  const popover = usePopover();
+  const [activeBreakdownFilter, setActiveBreakdownFilter] = useState<string>(
+    ""
+  );
+  const [smallMultiplesDialogOpen, setSmallMultiplesDialogOpen] = useState(
+    false
+  );
+
+  const dialogElementRef = useRef<HTMLElement>(null);
+  useEffect(() => {
+    if (smallMultiplesDialogOpen) {
+      const { current: dialogElement } = dialogElementRef;
+      if (dialogElement !== null) {
+        dialogElement.focus();
+      }
+    }
+  }, [smallMultiplesDialogOpen]);
```

Comment:
You can probably extract this into a simple hook `[isOpen, setIsOpen] = useAutoFocusDialog()`